### DISCUSSION TITLE
Videos uploaded upside down

### DISCUSSION THREAD

**Discussion by slitzer** [mediacms-io/mediacms#868] [Q&A]
**Describe the issue**
After uploading a video .mp4 (1,379,642 KB)
No issues with audio, butr video is upside down

**To Reproduce**
Steps to reproduce the issue:
1. Upload video
2. Completed
3. Upside down

**Expected behavior**
Video right side up, or option to rotate

**Screenshots**
![image](https://github.com/mediacms-io/mediacms/assets/12588795/5763f66e-15f5-446b-83be-a4ee79eb1ced)

**Environment (please complete the following information):**
 - OS: [e.g. Debien 10 Linux]
 - Installation method: [Docker]
 - Browser, Brave

**Additional context**
Does not happen to all videos

**Comment by mgogoulos** [comment]
Hi, I don't think there's anything that can be done here, if you don't upload to the actual video!
Looks strange indeed, haven't seen it myself happening

**Comment by slitzer** [comment]
Sorry, not sure what you mean upload to the actual video?

To note the video is not normally upside down, palyback if fine outsite mediacms, and works fine on Emby, Filebrowser

**Reply by mgogoulos** [reply]
Moving to discussions. It's not something I can reproduce, so if you can upload it on https://demo.mediacms.io/ for example and see the problem happening there, maybe I can have a look. Maybe if you can take a small part of it and upload it. Since the transcoding of the video is handled by ffmpeg, this seems something that ffmpeg creates (based on the settings we are providing), but it's impossible to assess what is happening without reproducing the problem!

**Comment by candrews67** [comment]
I have same issue with videos and pictures.

I updated a picture to the demo site, https://demo.mediacms.io/view?m=MuFkFNTSG .

thanks!!

**Comment by mgogoulos** [comment]
I've seen it too in images, I believe this is something with the imagekit processing and resizing. @candrews67 thanks for uploading the file so we can debug

**Comment by candrews67** [comment]
you are very welcome.if you need more please let me know On Sep 27, 2024 4:22 AM, Markos Gogoulos ***@***.***> wrote:
I've seen it too in images, I believe this is something with the imagekit processing and resizing. @candrews67 thanks for uploading the file so we can debug
